src/css/globals/vars.css

```diff
@@ -64,7 +64,7 @@
   --z-gdpr: 20;
   --z-modal-header: 16;
   --z-modal: 15;
-  --z-nav-mobile: 14;
+  --z-nav-mobile: 1000;
```

Comment:
Why did this have to bump up to override the other ones?  Did you test if search or whatever is a modal still works with this change?

Reply:
As of today, we can't test the search bar unless we build this on stgx.

Reply:
Ah, right.  Do what you think is best, then.